File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L3/03/Kasus3.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function main() {
  // Prompt the user to enter weight in pounds
  rl.question('Enter weight in pounds: ', (weightStr) => {
    const weight = parseFloat(weightStr);

    // Prompt the user to enter height
    rl.question('Enter feet: ', (feetStr) => {
      const feet = parseFloat(feetStr);
      rl.question('Enter inches: ', (inchesStr) => {
        const inches = parseFloat(inchesStr);

        const height = feet * 12 + inches;

        // Compute BMI
        const bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));

        // Display result
        console.log(`BMI is ${bmi}`);
        if (bmi < 18.5) {
          console.log('Underweight');
        } else if (bmi < 25) {
          console.log('Normal');
        } else if (bmi < 30) {
          console.log('Overweight');
        } else {
          console.log('Obese');
        }

        rl.close();
      });
    });
  });
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L3/03/Kasus3.kt
package IR

import java.util.Scanner

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
fun main() {
    val input = Scanner(System.`in`)

    // Prompt the user to enter weight in pounds
    print("Enter weight in pounds: ")
    val weight = input.nextDouble()

    // Prompt the user to enter height
    print("Enter feet: ")
    val feet = input.nextDouble()
    print("Enter inches: ")
    val inches = input.nextDouble()

    val height = feet * 12 + inches

    // Compute BMI
    val bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

    // Display result
    println("BMI is $bmi")
    when {
        bmi < 18.5 -> println("Underweight")
        bmi < 25 -> println("Normal")
        bmi < 30 -> println("Overweight")
        else -> println("Obese")
    }
}

